x/consensus/keeper/keeper.go

```diff
@@ -34,26 +34,28 @@ func (k *Keeper) GetAuthority() string {
 
 // Get gets the consensus parameters
 func (k *Keeper) Get(ctx sdk.Context) (*cmtproto.ConsensusParams, error) {
```

Comment:
all functions can now take a context.Context here. this would further cleanup things, we can do it in a follow up pr but might make sense to do the cleanup all at once

Reply:
on it ser